mesonbuild/build.py

```diff
@@ -252,6 +252,7 @@ def __init__(self, environment: environment.Environment):
         self.environment = environment
         self.projects = {}
         self.targets: 'T.OrderedDict[str, T.Union[CustomTarget, BuildTarget]]' = OrderedDict()
+        self.targetnames: T.Set[(str, str)] = set() # Set of executable names and their subdir
```

Comment:
Other places in the file spell this as T.Tuple[str, str], I suspect mypy will complain.

Reply:
Yeah as far as the type annotations are concerned this is just Set[str, str] I think.

Reply:
Ah yup.